models/Bitmap/Bitmap.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with
+ * the License. A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+const uuid = require('uuid-v4');
+const fs = require('fs-extra');
+const md5File = require('md5-file');
+const Layer = require('../Layer');
+const Rect = require('../Rect');
+const Style = require('../Style');
+const { STORAGE_DIR, STORAGE_IMG_DIR } = require('../../utils/paths');
+
+/**
+ * A bitmap layer is used for images
+ * @extends Layer
+ *
+ */
+class Bitmap extends Layer {
+  /**
+   *
+   * @param {Object} args
+   * @param {String} args.filePath Local path of the image
+   * @param {String} args.image The path of the image
+   * @param {Object} args.frame Sent to {@link Rect}
+   * @param {Object} args.style Sent to {@link Style}
+   * @param {Bitmap.Model} json
+   */
+  constructor(args = {}, json) {
+    super(args, json);
+    if (!json) {
+      const id = args.id || uuid().toUpperCase();
+      Object.assign(this, Bitmap.Model, {
+        do_objectID: id,
+        name: args.name || id,
+        frame: new Rect(args.frame),
+        style: new Style(args.style),
+        layers: args.layers || [],
+      });
+      const fileHash = md5File.sync(args.filePath);
+      this.image._ref = `images/${fileHash}.png`;
+      if (!fs.existsSync(STORAGE_IMG_DIR)) {
+        fs.mkdirSync(STORAGE_IMG_DIR, { recursive: true });
+      }
+      fs.copyFile(args.filePath, `${STORAGE_DIR}/${this.image._ref}`);
```

Comment:
Should this be a sync method (copyFileSync)? Also should this be `STORAGE_IMG_DIR` instead of `STORAGE_DIR`

Reply:
I think I already set `image._ref` to be `image/filename.png` here https://github.com/amzn/sketch-constructor/pull/48/files#diff-1a679271088cc5765b05f04b930eb5fbR49. Because Sketch expects `_ref` to include the `image/` dir.

Reply:
Ah ok got it. Thanks!